pkg/internal/gcp/gcp.go

```diff
@@ -0,0 +1,164 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gcp
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"strings"
+
+	"google.golang.org/api/secretmanager/v1"
```

Comment:
Isn't https://github.com/googleapis/google-cloud-go the more official SDK. https://github.com/googleapis/google-api-go-client is auto-gen'd.

Reply:
* [docs](https://pkg.go.dev/cloud.google.com/go/secretmanager/apiv1#pkg-overview) show `cloud.google.com/go/secretmanager/apiv1` is autogenerated
* [docs](https://pkg.go.dev/google.golang.org/api/secretmanager/v1#pkg-overview) show `google.golang.org/api/secretmanager/v1` is not